src/index.ts

```diff
@@ -1,5 +1,50 @@
-"use strict";
+import { SessionMiddlewareFactory } from "./SessionMiddlewareFactory";
+import { SessionHandlerConfig } from "./SessionHandlerConfig";
+import { SessionStore } from "./session/SessionStore";
+import { Cache } from "./cache/Cache";
+import { RequestHandler } from "express";
 
-import SessionMiddlewareFactory from "./SessionMiddlewareFactory";
 
-export = SessionMiddlewareFactory;
+export interface CHSessionService {
```

Comment:
What is the purpose of this service? Does it have one clearly defined purpose of existence?